mlir/utils/performance/ck-benchmark-driver/CMakeLists.txt

```diff
@@ -1,17 +1,19 @@
-find_package(composable_kernel CONFIG 1.0.0 COMPONENTS device_operations)
-if(NOT composable_kernel_FOUND)
-  message(WARNING "Could not find composable-kernels, not building a CK benchmark driver")
-endif()
+if(MLIR_USE_CK)
```

Comment:
I'd like to move this `if` one level up, into the `CMakeLists.txt` for `performance/`